<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCU_Studio</class>
 <widget class="QMainWindow" name="MCU_Studio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mcu Studio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Res.qrc">
    <normaloff>:/img/app.ico</normaloff>:/img/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 254, 221);</string>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>File Name</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
            <item>
             <widget class="QTextEdit" name="textEdit">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 254, 221);</string>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 254, 221);ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPlaster"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build(&amp;B)</string>
    </property>
    <addaction name="actionBuildProject"/>
    <addaction name="actionBuild_Project_All"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug(&amp;D)</string>
    </property>
    <addaction name="actionStart_Debug"/>
    <addaction name="actionStop_Debug"/>
    <addaction name="actionPause_Debug"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <addaction name="actionSerial_Communication"/>
    <addaction name="separator"/>
    <addaction name="actionGit"/>
    <addaction name="separator"/>
    <addaction name="actionC"/>
    <addaction name="actionJava"/>
    <addaction name="actionC_2"/>
    <addaction name="separator"/>
    <addaction name="actionWorkspace"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Screen"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window(&amp;W)</string>
    </property>
    <addaction name="actionClose_Windows"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionWeb_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuDebug"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionBuildProject"/>
   <addaction name="separator"/>
   <addaction name="actionSerial_Communication"/>
   <addaction name="separator"/>
   <addaction name="actionWorkspace"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/edit.svg</normaloff>:/img/edit.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Esc</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/printer.svg</normaloff>:/img/printer.svg</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBuildProject">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/debug.png</normaloff>:/img/debug.png</iconset>
   </property>
   <property name="text">
    <string>Build Project</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionBuild_Project_All">
   <property name="text">
    <string>Build Project All</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/copy.png</normaloff>:/img/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPlaster">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/paste.png</normaloff>:/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>Plaster</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/cut.png</normaloff>:/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionStart_Debug">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/debug-run-cursor.png</normaloff>:/img/debug-run-cursor.png</iconset>
   </property>
   <property name="text">
    <string>Start Debug</string>
   </property>
  </action>
  <action name="actionStop_Debug">
   <property name="text">
    <string>Stop Debug</string>
   </property>
  </action>
  <action name="actionPause_Debug">
   <property name="text">
    <string>Pause Debug</string>
   </property>
  </action>
  <action name="actionSerial_Communication">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/com.png</normaloff>:/img/com.png</iconset>
   </property>
   <property name="text">
    <string>Serial Communication</string>
   </property>
  </action>
  <action name="actionGit">
   <property name="text">
    <string>Git</string>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>C++</string>
   </property>
  </action>
  <action name="actionJava">
   <property name="text">
    <string>Java</string>
   </property>
  </action>
  <action name="actionC_2">
   <property name="text">
    <string>C</string>
   </property>
  </action>
  <action name="actionClose_Windows">
   <property name="text">
    <string>Close Windows</string>
   </property>
  </action>
  <action name="actionWeb_Help">
   <property name="text">
    <string>Web Help</string>
   </property>
  </action>
  <action name="actionWorkspace">
   <property name="icon">
    <iconset resource="Res.qrc">
     <normaloff>:/img/settings.svg</normaloff>:/img/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Workspace</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Screen">
   <property name="text">
    <string>New Screen</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
